Doc/tools/templates/indexcontent.html

```diff
@@ -7,62 +7,62 @@ <h1>{{ docstitle|e }}</h1>
   <p>
   {% trans %}Welcome! This is the official documentation for Python {{ release }}.{% endtrans %}
   </p>
-  <p><strong>{% trans %}Parts of the documentation:{% endtrans %}</strong></p>
+  <p><strong>{% trans %}Documentation sections:{% endtrans %}</strong></p>
   <table class="contentstable" align="center"><tr>
     <td width="50%">
       <p class="biglink"><a class="biglink" href="{{ pathto("whatsnew/" + version) }}">{% trans %}What's new in Python {{ version }}?{% endtrans %}</a><br/>
-        <span class="linkdescr"> {% trans whatsnew_index=pathto("whatsnew/index") %}or <a href="{{ whatsnew_index }}">all "What's new" documents</a> since 2.0{% endtrans %}</span></p>
+         <span class="linkdescr"> {% trans whatsnew_index=pathto("whatsnew/index") %}or <a href="{{ whatsnew_index }}">All "What's new" documents since Python 2.0</a>{% endtrans %}</span></p>
```

Comment:
```suggestion
         <span class="linkdescr"> {% trans whatsnew_index=pathto("whatsnew/index") %}Or <a href="{{ whatsnew_index }}">all "What's new" documents since Python 2.0</a>{% endtrans %}</span></p>
```

So instead of:

> or [All "What's new" documents since Python 2.0](https://cpython-previews--116914.org.readthedocs.build/en/116914/whatsnew/index.html)

It looks like:

> Or [all "What's new" documents since Python 2.0](https://cpython-previews--116914.org.readthedocs.build/en/116914/whatsnew/index.html)